framework/include/ble_manager/ble_manager.h

```diff
@@ -90,3 +85,7 @@ ble_result_e ble_manager_deinit(void);
  *
  ****************************************************************************/
 ble_result_e ble_manager_get_mac_addr(uint8_t mac[BLE_BD_ADDR_MAX_LEN]);
+
```

Comment:
Please include `#include <stdbool.h>` for `bool` type.

Reply:
OK, I will add it